Neos.ContentRepository.Core/Classes/CommandHandler/ControlFlowAwareCommandHandlerInterface.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Neos\ContentRepository\Core\CommandHandler;
+
+use Neos\ContentRepository\Core\CommandHandlingDependencies;
+use Neos\ContentRepository\Core\EventStore\EventsToPublish;
+use Neos\ContentRepository\Core\EventStore\EventsToPublishFailed;
+
+/**
+ * Common interface for all Content Repository command handlers
```

Comment:
interface for all Content Repository command handlers which need to interact with the control flow
of event publishing, e.g. for error handling.
This is used in combination with CommandSimulator for Workspace partial Publish and rebase.

Reply:
still makes sense as additional context comment